CHANGELOG.md

```diff
@@ -5,13 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.12.0] - 2023-11-01
+
+### Changed
+
+- YPP requirements are now fetched from yt-synch
+- Changed rewards for higher tiers
+- Moved YPP daily quota to config
```

Comment:
Aren't we also including the top referrers feature too?

Reply:
no, as we have some CR fixes to make and Kuba is off today it'll be postponed